portal-api/src/v1/users/validation/rules/username-and-email-must-match.js

```diff
@@ -0,0 +1,16 @@
+module.exports = (user, change) => {
+  const error = [
+    {
+      email: {
+        order: '1',
+        text: 'Username and email must match',
+      },
+    },
+  ];
+
+  if (change?.username !== change?.email) {
+    return error;
+  }
+
+  return [];
+};
```

Comment:
Not a blocker - probably tech debt. Some documentation would be good here, also - probably should be done in one go, but I find it easier to read/maintain if we have a function name at the top and `module.exports` at the bottom of the file, e.g

```js
module.exports = usernameAndEmailMustMatchRule;
```

Reply:
Agree - will add to next ticket